README.md

```diff
@@ -22,10 +22,32 @@ The full reference documentation for rules is at https://haskell.build.
 
 ## Setup
 
-You'll need [Bazel >= 0.27][bazel-getting-started] installed.
+You'll need [Bazel >= 0.29][bazel-getting-started] installed.
 
 If you are on NixOS, skip to the [Nixpkgs](#Nixpkgs) section.
 
+### System dependencies
+
+On a Unix system you will need the following tools installed.
+
+- `gcc`
+- `libffi`
+- `libgmp`
+- `libtinfo5`
+- `make`
+- `python3`
+
+On Ubuntu you can obtain them by installing the following packages.
+
+```
+build-essential libffi-dev libgmp-dev libtinfo5 libtinfo-dev python
```

Comment:
On Ubuntu, `python` means `python2`. `python3` seems to be already installed (and looking at the shebangs in the repo, I assume we also need it ;-)).

Should we explicitly list Python 2 and Python 3 here? Or how do the bindists work? Do they ship their own Python?

Reply:
@flokli I took that list of Ubuntu packages from here https://github.com/tweag/rules_haskell/pull/1162#discussion_r350675766.
I've replaced `python` with `python3`.

Reply:
This got out of sync with `docs/haskell.rst`. Can we only describe it in `docs/haskell.rst`, and refer to that file in here?

Reply:
Good point, done.